src/hiero_sdk_python/tokens/token_airdrop_pending_record.py

```
@@ -30,24 +33,26 @@ def _from_proto(cls, proto: transaction_record_pb2.PendingAirdropRecord) -> "Pen
 
         """
         return transaction_record_pb2.PendingAirdropRecord(
             pending_airdrop_id=self.pending_airdrop_id._to_proto(),
-            pending_airdrop_value=basic_types_pb2.PendingAirdropValue(amount=self.amount)
+            pending_airdrop_value=basic_types_pb2.PendingAirdropValue(
+                amount=self.amount
+            ),
         )
-    
+
     def __str__(self):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)